relnotes/python3-migration.md

```diff
@@ -0,0 +1,3 @@
+### Port to Python3
+
+`git-hub` has been ported to **Python 3** and can no longer be run with **Python 2.x**.
```

Comment:
Migration instructions should include... migration instructions :stuck_out_tongue_closed_eyes: 

Even when it might sound obvious, I would be explicit about what the user have to take care to make sure an upgrade won't break their installation. Something as basic as:

> Make sure you have Python3 installed. Check the Python version using `python --version` or check if the `python3` program exists. If you need to install it, use `apt install python3` in Debian based distributions for example.

Reply:
Also the title of the entry might be better to be more explicit, like `Python version 3 required`.